tests/integration/utils/test_sparams.py

```
@@ -0,0 +1,614 @@
+    )
+
+
+@pytest.fixture(scope="session")
+def two_port_setup():
+    """2 named input ports (P1, P2) + 2 named output ports (P3, P4)."""
+    xs = [0.88e-6, 1.1e-6, 1.32e-6, 1.54e-6]
+    cy, cz = 0.8e-6, 0.8e-6
+    names = ["P1", "P2", "P3", "P4"]
+    ports = [
+        PortSpec(center=(x, cy, cz), axis=0, direction="+", width=_PORT_WIDTH, height=_PORT_HEIGHT, name=n)
+        for x, n in zip(xs, names)
```

Ruff (0.15.10)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)